docker-compose.yml

```diff
@@ -15,11 +22,14 @@ hydroshare:
     POSTGIS_DB: postgres
     POSTGIS_USER: postgres
     PGPASSWORD: postgres
+    SOLR_HOST: solr
   volumes:
     # hydroshare repository
-    - ".:/home/docker/hydroshare"
+    - "/home/hydro/hydroshare:/home/docker/hydroshare"
     # pycharm debugging
-    #- "/home/hydro/pycharm-debug:/home/docker/pycharm-debug"
+    # - "/home/hydro/pycharm-debug:/home/docker/pycharm-debug"
+    # log files
+    - "/home/hydro/hydroshare/log:/var/log/hydroshare"
```

Comment:
These three lines don't have to be changed/added either, since the log volume mapping will be added from docker-compose.template when running hsctl.